scss/_layouts_docs.scss

```diff
@@ -0,0 +1,31 @@
+@mixin vf-l-docs {
+  .l-fixed-width--with-sidebar {
+    @extend %vf-row;
+  }
+
+  .l-sidebar,
+  .l-content {
```

Comment:
These should be nested in the main layout class to avoid cross layout overspill

Reply:
Yes, I'm thinking how to do that.
Technically should be `l-fixed-width__sidebar`. But because sidebar works only for `--with-sidebar`, should it be `l-fixed-width--with-sidebar__sidebar` - this feels overkill and wrong.

So maybe using `--with-sidebar` is wrong. Maybe it should just be a top level layout, not a modifier?
`l-fixed-width-with-sidebar` `l-fixed-width-with-sidebar__sidebar` `l-fixed-width-with-sidebar__content`?

Avoiding calling it `l-docs` to have more generic name that is not tied to one use case makes these class names very verbose and long. I'm struggling with giving them short and descriptive names.